util.go

```diff
@@ -208,3 +208,26 @@ func getBinaryFileName(filePath string, dumpType configureType) string {
 
 	return path.Join(filePath, type2name[dumpType]+"."+binarySuffix)
 }
+
+func writeFile(data bytes.Buffer, dumpType configureType, dumpOpts *DumpOptions, logf logFncTyp) {
```

Comment:
it's better to return an error here and log it on the caller side instead of passing the `logf` argument.

Reply:
ok